DeDRM_plugin/ignoblekeyGenPassHash.py

```diff
@@ -214,9 +139,8 @@ def generate_key(name, ccn):
 
     name_sha = hashlib.sha1(name).digest()[:16]
     ccn_sha = hashlib.sha1(ccn).digest()[:16]
-    both_sha = hashlib.sha1(name + ccn).digest()
-    aes = AES(ccn_sha, name_sha)
-    crypt = aes.encrypt(both_sha + (b'\x0c' * 0x0c))
+    both_sha = hashlib.sha1()(name + ccn).digest()
```

Comment:
This line seems to be invalid and causes an exception, the first set of brackets should probably be removed.

Reply:
Oops - fixed